presto-orc/src/main/java/com/facebook/presto/orc/OrcReaderConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+public class OrcReaderConfig
```

Comment:
I can see you are trying to abstract the reader params into this. That's good start. In order to make it holistic, here is my suggestion:
1. Move `ReaderAttributes` in presto-raptor as a common class in presto-orc. Do not inject it. Call it `OrcReaderOptions` just like what we have been doing with `OrcWriterOptions`.
2. Put `zstdJniDecompressionEnabled` in `ReaderAttributes`
3. Clean up `OrcReader` interface with `OrcReaderOptions`

Reply:
ReaderAttributes seems to have additional configuration apart from what OrcReader requires. For example, streamBufferSize and lazyReadSmallRanges don't seem to be applicable to the OrcReader. 

Alternatively, I could create OrcReaderOptions with the minimum set of configuration and ReaderAttributes can stay as is.

Reply:
Will compose OrcReader options with ReaderAttributes.

Reply:
Reader Attributes does not contain Max Block Size and moving that there is additional work. So I have left ReaderAttributes as is for now.